bots/pr/src/main/java/org/openjdk/skara/bots/pr/CommandInvocation.java

```diff
@@ -10,13 +11,15 @@
     private final CommandHandler handler;
     private final String name;
     private final String args;
+    private final Comment comment;
 
-    CommandInvocation(String id, HostUser user, CommandHandler handler, String name, String args) {
+    CommandInvocation(String id, HostUser user, CommandHandler handler, String name, String args, Comment comment) {
```

Comment:
Think you can drop this parameter if you drop the exclude param!

Reply:
Yep, fixed in 7b17768